tests/unit/src/test/scala/tests/NewFileTemplateSuite.scala

```diff
@@ -0,0 +1,33 @@
+package tests
+
+import munit.ScalaCheckSuite
+import scala.meta.internal.metals.NewFileTemplate
+import org.scalacheck.Gen
+import org.scalacheck.Prop.forAll
+
+class NewFileTemplateSuite extends BaseSuite with ScalaCheckSuite {
+
+  test("it requires exactly one cursor marker") {
```

Comment:
Should we stick to short names as in other suites?

Reply:
Ah, absolutely, I completely forgot. Fixed